### PR TITLE
Clarified how to upgrade 1.7.X to a 2.X.X.

### PR DESCRIPTION
**Pull request by Darkbelg**
Transporteo requires the minimum version to be 2.0.x. So you have to install 2.0.x first and then progressively update. Unless somebody can tell me you can go from 2.0 to 2.3 directly ?

**Description**

**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Technical Review and 2 GTM        | Todo
| English Review and 1 GTM          | Todo

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: migrate_pim/upgrade_major_version.rst [hunk 1]
```diff
@@ -4,14 +4,15 @@ How to upgrade to a major version?
 We release a major version each year.
 These new major versions bring new features and larger changes to answer clients growing needs.
 
-Migrate from 1.7 to 2.0
+Migrate from 1.7 to 2.X
 ------------------------------
 
-To migrate from 1.7 to 2.0, we recommend the use of our brand new migration tool `Transporteo`_.
+To migrate from 1.7 to 2.X.X you need to install version 2.0.X first.
+Then we recomend using the migration tool `Transporteo`_.
```
Comment (rybus) [inline migrate_pim/upgrade_major_version.rst]: I feel like this sentence makes us think we first have to go to 2.0.x and then use transporteo.

Comment (rybus) [reply migrate_pim/upgrade_major_version.rst]: It's more that you first need to use transporteo to be on 2.0.x and THEN you'll have to do individual migrations following the UPGRADE-2.1, -2.2 and -2.3.md.

**Comment by Darkbelg** [comment]
I have edited the file to incorporate the feedback i don't know what button to press to get a new review.

File: migrate_pim/upgrade_major_version.rst (hunk 1 above)
Comment (Darkbelg) [reply migrate_pim/upgrade_major_version.rst]: Well then you can change it if you think you can clarify it better. You know what i mean.
Maybe we could use local instance and a remote instance for the different akeneo versions ?

Or maybe say first create a new akeneo install with version 2.0.X.

**Pull request closed without merging**